scripts/protocol-dts-generator.ts

```diff
@@ -48,18 +40,17 @@ const emitCloseBlock = (closeChar = '}') => {
 }
 
 const emitHeaderComments = () => {
-    emitLine("/**********************************************************************")
-    emitLine(" * Auto-generated by protocol-dts-generator.ts, do not edit manually. *")
-    emitLine(" **********************************************************************/")
+    emitLine('/**********************************************************************')
+    emitLine(' * Auto-generated by protocol-dts-generator.ts, do not edit manually. *')
+    emitLine(' **********************************************************************/')
     emitLine()
 }
 
-const emitModule = (moduleName:string, domains: P.Domain[]) => {
+const emitModule = (moduleName: string, domains: P.Domain[]) => {
```

Comment:
ha! spaces! We should just get this tslinted so you don't have to curse me everytime you see a missing space.

Reply:
> We should just get this tslinted

agreed :)